### PR TITLE
Promote rabbitmqexporter to alpha

### PR DESCRIPTION
**Pull request by swar8080**
**Description:**
Promotes the RabbitMQ exporter to alpha

**Testing:**
Manual testing and integration tests implemented in a previous PR

### CODE REVIEW

**Comment by mx-psi** [comment]
Should this be a draft PR based on the title? If so, can you mark it as draft using the link below the reviewers list? 
image

**Review by fatsheep9146 — APPROVED** [review]
LGTM!
BTW, I think https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml should also be updated.

**Review by codeboten — COMMENTED** [review]
@swar8080 when you have a second to rebase, we can get this merged shortly after

**Review by codeboten — APPROVED** [review]
Pinging other code owner to confirm: @atoulme

**Comment by swar8080** [comment]
Hi @atoulme, this seems ready to merge once you approve

**Review by atoulme — APPROVED** [review]

File: exporter/rabbitmqexporter/README.md
Comment (atoulme) [inline exporter/rabbitmqexporter/README.md:11]: Remember to add it in opentelemetry-releases

**Review by atoulme — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**
